File: branches/junit/pysys-examples/junit/testcases/JUnit_test_003/Input/TestBroken.java
import org.junit.*;
import static org.junit.Assert.*;

import static totest.Broken.*;

public class TestBroken {
	@Test
	public void testAddOne() {
		// This happens to work, but by accident
		assertEquals(brokenAdder(5,1), 6);
	}

	@Test
	public void testAddTwo() {
		// This should fail
		assertEquals(brokenAdder(3,2), 5);
	}
}

File: tags/PySys_0_9_0/pysys-examples/pyunit/testcases/PyUnit_test_003/Input/testBroken.py
import broken
import unittest

class TestBroken(unittest.TestCase):

	def testAddOne(self):
		# This happens to work, but by accident
		self.assertEquals(broken.brokenAdder(5,1), 6)

	def testAddTwo(self):
		# This should fail
		self.assertEquals(broken.brokenAdder(3,2), 5)

